.circleci/config.yml

```diff
@@ -46,7 +47,9 @@ before_script: &before_script
         paths:
           - coq/
 
-  environment: *envvars
+  environment:
+    <<: *envvars
+    NATIVE_COMP: "yes"
```

Comment:
Why this change?

Reply:
Otherwise the image vars are not picked up in subjobs.